test/e2e/e2e-service-provider/pom.xml

```diff
@@ -44,6 +44,11 @@
             <artifactId>h2</artifactId>
             <version>${h2.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.skywalking</groupId>
+            <artifactId>apm-toolkit-trace</artifactId>
+            <version>8.0.0-SNAPSHOT</version>
```

Comment:
This will be hard to maintain, though I didn't figure out a better way

Reply:
Well, it also breaks the CI because we don't have this version when building this service

Reply:
So @mrproliu you have to copy class `org.apache.skywalking.apm.toolkit.trace.TraceContext` into e2e folder, and remove this dependency,

Reply:
Yes, I have tried on this. But When I copy on this, the IDE will give me an alert, because it also a source directory. Should I need to add the e2e project to a new IDE window?
![image](https://user-images.githubusercontent.com/3417650/79137265-f71b5300-7de4-11ea-9bc6-05d747d13597.png)

Reply:
> Yes, I have tried on this. But When I copy on this, the IDE will give me an alert, because it also a source directory. Should I need to add the e2e project to a new IDE window?
> ![image](https://user-images.githubusercontent.com/3417650/79137265-f71b5300-7de4-11ea-9bc6-05d747d13597.png)

Copy the file out of the IDEA, i.e. in the `Finder` or `Terminal`, or open the `e2e` in a separate IDEA window and copy between the two windows (`apm` and `e2e`)